### PR TITLE
correctly align read receipts to state events in bubble layout

### PR DESCRIPTION
**Pull request by justjanne**
Type: Defect
Closes: https://github.com/vector-im/element-web/issues/21899

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * correctly align read receipts to state events in bubble layout ([\#8419](https://github.com/matrix-org/matrix-react-sdk/pull/8419)). Fixes vector-im/element-web#21899. Contributed by @justjanne.

Preview: https://pr8419--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Review by robintown — APPROVED** [review]
Looks good!

File: res/css/views/rooms/_EventBubbleTile.scss [hunk 1]
```diff
@@ -450,7 +450,7 @@ limitations under the License.
     .mx_ReadReceiptGroup {
         position: absolute;
         right: -78px; // as close to right gutter without clipping as possible
-        bottom: 0;
+        bottom: -1px;
```
Comment (turt2live) [inline res/css/views/rooms/_EventBubbleTile.scss:453]: This needs a comment for what 1px represents.

**Review by turt2live — DISMISSED** [review]
also screenshots please, so we can have a reference point in 4 years when it breaks again :p

**Comment by justjanne** [comment]
## In Action

* New correct alignment  
Bubble Status Read Receipt Alignment
* New correct alignment as false-color image  
Bubble Status Read Receipt Alignment in False Color

**Pull request merged**